quipucords/api/details_report/tests_details_report.py

```diff
@@ -178,3 +181,42 @@ def test_csv_renderer(self):
         # pylint: disable=line-too-long
         expected = 'Report ID,Report Type,Report Version,Number Sources\r\n1,details,%s,1\r\n\r\n\r\nSource\r\nServer Identifier,Source Name,Source Type\r\n%s,test_source,network\r\nFacts\r\n\r\n' % (self.report_version, self.server_id)  # noqa
         self.assertEqual(csv_result, expected)
+
+    ##############################################################
+    # Test Json Gzip Render
+    ##############################################################
+    def test_json_gzip_renderer(self):
```

Comment:
Let's add tests for report ID not found.  We should have these for all content types we return (should means ideally)